halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/v2/KubernetesV2OrcaService.java

```diff
@@ -34,4 +46,40 @@
   public ServiceSettings defaultServiceSettings() {
     return new Settings();
   }
+
+  public static class Builder extends SpringService.Builder<KubernetesV2OrcaService,Builder> {
```

Comment:
This builder seems duplicated for each service. Previously the only differences between services were tightly expressed here. Is it possible to remove/refactor this?